R/nhsbsaShinyR.R

```diff
@@ -0,0 +1,20 @@
+#' \code{nhsbsaShinyR} package
+#'
+#' Template app using `golem` for NHSBSA DALL `shiny` apps.
+#'
+#' @docType package
+#' @name nhsbsaShinyR
+#'
+#' @importFrom rlang := .data
```

Comment:
Adding this file (package documentation) as a convenient place to add imports and global variables.